airflow/providers/amazon/aws/sensors/batch.py

```diff
@@ -41,6 +43,7 @@ class BatchSensor(BaseSensorOperator):
     :param job_id: Batch job_id to check the state for
     :param aws_conn_id: aws connection to use, defaults to 'aws_default'
     :param region_name: aws region name associated with the client
+    :param deferrable: Run sensor in the deferrable mode.
```

Comment:
Why aren't `poke_interval` and `max_retries` here?

Reply:
Fixed in [388cbaa](https://github.com/apache/airflow/pull/30279/commits/388cbaa3572fd7e46985e33edadd92ec372d7169)